src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -507,6 +507,34 @@ def set_browser_implicit_wait(self, seconds):
         implicit_wait_in_secs = robot.utils.timestr_to_secs(seconds)
         self._current_browser().implicitly_wait(implicit_wait_in_secs)
 
+    def get_browser_info(self, type='object'):
```

Comment:
Well, my point of view as a Tester I just usually want to get the Browser name and version. I thought i was more natural to get browser information. I will rename it to `get_browser_capabilities` because they are not necessarily the "desired" capabilities.